GoogleForCreators/web-stories-wp
# Elements: Add default positioning for text

## Summary
Adds default positioning for text -- vertically centered and 40 "pixels" from the left.


## Relevant Technical Choices
Note that this does not include staggering so it's expected that adding multiple elements in a row will add them exactly to the same place (see #1206).


## To-do



## User-facing changes
New text is always added vertically middle and 40 "pixels" from the left side.


## Testing Instructions
I
1. Add any Text element
2. Verify that it is centered vertically on the Page
3. Verify that it has room between the left Page edge.

II
1. Add Text element from the `+` button
2. Verify that it's text-align is center.


---



Fixes #1204
